Verkiezingen_functies.py

```diff
@@ -101,8 +100,78 @@ def perc_ongeldig_gemeente(uitslagenDF, gemeente):
     
     return percDF
     
-    
-  
+def provincie_stemmen(provincie):   #deze functie pakt de lijst van gemeentes per provincie en de normale dataframe en voegt ze vervolgens samen op gemeentenaam als key.
+ #                                  #daarna wordt er op provincie geflitered en geeft df_prinvcie2 alleen de resultaten uit de normale dataframe terug voor die provincie
+    uitslagenDF = pd.read_csv(r'Uitslag_alle_gemeenten_TK20210317.csv', sep=';')
+    uitslagentweeDF = pd.read_csv(u'Gemeenten alfabetisch 2019.csv', sep=',')
+    #print(uitslagentweeDF)
+    prov_gem_df = pd.DataFrame(data=list(uitslagentweeDF['Provincienaam']), columns=['Provincienaam'], index=list(uitslagentweeDF['Gemeentenaam']))
+    prov_gem_df.index.name = 'Gemeentenaam'
+    #print(uitslagentweeDF)
+    #print(prov_gem_df)
+    df_merge = pd.merge(uitslagenDF, uitslagentweeDF, how='left', left_on=['RegioNaam'], right_on=['Gemeentenaam'])
+    #print(df_met_provincies)
```

Comment:
Ik zou toch willen voorstellen om het inladen van de csv's en het mergen van de df's in province_als_landelijk te doen, nu doe je het daar 12 keer in een loop terwijl het maar 1 keer hoeft in het begin. Dan zou je zelfs deze provincie_stemmen op kunnen heffen en alles onder provincie_als_landelijk kunnen doen. Op zich werkt het zo natuurlijk, het is voornamelijk een suggestie voor een stukje efficiëntie.